app/views/teacher_interface/registration_number/edit_ghana.html.erb

```diff
@@ -0,0 +1,77 @@
+<% content_for :page_title, title_with_error_prefix(t("application_form.tasks.items.registration_number.ghana"), error: @form.errors.any?) %>
+<% content_for :back_link_url, back_history_path(default: teacher_interface_application_form_path) %>
+
+<%= form_with model: @form, url: %i[registration_number teacher_interface application_form] do |f| %>
+  <%= f.govuk_error_summary %>
+
+  <div class="govuk-form-group <%= 'govuk-form-group--error' if @form.errors.present? %>">
```

Comment:
Note that this form group had to be done in a custom way since this form component does not exist. However, we have reused the date field component and ensured it matches the design system.